todo-app/src/TodoItem.js

```diff
@@ -6,15 +6,23 @@ import './index.css';
 export default class TodoItem extends React.Component {
 
   onDelete = () => this.props.onDelete(this.props.todoItem.id)
+
+  onDone = () =>  this.props.onDone(this.props.todoItem.id)
```

Comment:
What's happening here? You're redeclaring `onDone` inside your child component `TodoItem`, which is fine, but you never use `this.onDone` - you're still using the `onDone` function from the `props` with an inline anonymous function:

    onClick={() => onDone(todoItem.id)}

Both ways are totally fine, but choose one. In this case both `this.onDelete` and `this.onDone` are never used.